deps/chakrashim/src/inspector/java-script-call-frame.cc

```diff
@@ -88,7 +92,8 @@ int JavaScriptCallFrame::contextId() const {
 
 bool JavaScriptCallFrame::isAtReturn() const {
   int index;
-  if (InspectorHelpers::GetIntProperty(m_callFrame, "index", &index) != JsNoError) {
+  if (jsrt::GetProperty(m_callFrame, jsrt::CachedPropertyIdRef::index,
+                        &index) != JsNoError) {
     assert(false);
     return false;
   }
```

Comment:
I actually just switched all these to CHAKRA_VERIFY_NOERROR which was the original intention, but at that point the code was separated and that macro wasn't available.